frontend/src/components/mine/ContactInfo/PartyRelationships/Contact.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import CustomPropTypes from "@/customPropTypes";
+
+import { DefaultContact } from "@/components/mine/ContactInfo/PartyRelationships/DefaultContact";
+import { EngineerOfRecord } from "@/components/mine/ContactInfo/PartyRelationships/EngineerOfRecord";
+import { Permittee } from "@/components/mine/ContactInfo/PartyRelationships/Permittee";
+
+const propTypes = {
+  partyRelationship: CustomPropTypes.partyRelationship.isRequired,
+  partyRelationshipTypeLabel: PropTypes.string.isRequired,
+  handleChange: PropTypes.func,
+  mine: PropTypes.object.isRequired,
+  openEditPartyRelationshipModal: PropTypes.func,
+  onSubmitEditPartyRelationship: PropTypes.func,
+  removePartyRelationship: PropTypes.func,
+  otherDetails: PropTypes.object,
+  isEditable: PropTypes.string,
+};
+
+const doNothing = () => {};
+
+const defaultProps = {
+  handleChange: () => {
```

Comment:
The component defaults to a mode that is does not have the ability to add or update a contact. The component functions as expected without them in this mode.

Reply:
Just to make sure my feedback was clear... I recognize that React/redux could be doing something clever with closures here that makes the outer function necessary, but it does seem really odd to me that this works:
```javascript
const defaultProps = {
  handleChange: () => {
    doNothing();
  },
};
```
And this doesn't:
```javascript
const defaultProps = {
  handleChange: doNothing,
};
```

Since they look like the same logic to me.